app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="main_activity_buttonInset">10dp</dimen>
+    <dimen name="textSize_strings_20sp">20sp</dimen>
```

Comment:
- Use always only one name style, snake_caso or camelCase, the resources will be con scnake_case.
- Use more representative names
- Do not put the specific dime in the title

Reply:
Done